Четвертое задание

yaoyao668 opened the pull request:
Здравствуйте, учитель, это четвертое задание, войдите в систему по электронной почте.

yaoyao668 committed:
Четвертое задание

vladefr97 commented on AuthService/app/main.py:
```diff
@@ -0,0 +1,15 @@
+
+app = FastAPI()
+
+# 包含路由
```
Пиши комментарии на английском

vladefr97 commented on AuthService/app/config.py:
```diff
@@ -0,0 +1,4 @@
+import os
+
+DATABASE_URL = os.getenv("DATABASE_URL", "postgresql://postgres:123456@localhost/User")
+SECRET_KEY = "926e3c3b282f22e6c09ed7f3824deedc37993d40e6f90c08c2bc07cbf710fca6"
```
Нельзя заливать в проект реальную секретную информацию

vladefr97 commented on AuthService/app/auth.py:
```diff
@@ -0,0 +1,90 @@
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+
+def create_access_token(data: dict, expires_delta: timedelta = None):
```
Желательно разнести все по отдельным файлам

yaoyao668 replied:
Здравствуйте, учитель, я изменил программу по мере необходимости, пожалуйста, проверьте ее.

yaoyao668 committed:
Пересмотренное четвертое задание

Здравствуйте учитель, я поменял комментарии на английский, удалил зашифрованную информацию из проекта, а маршруты разбил на разные файлы.
